test/go/data_iterator_test.go

```diff
@@ -164,6 +166,20 @@ func (this *DataIteratorTestSuite) completedTables() map[string]bool {
 	return this.di.StateTracker.Serialize(nil, nil).CompletedTables
 }
 
+func (this *DataIteratorTestSuite) TestDataIterationBatchSizePerTableOverride() {
+	for _, table := range this.tables {
+		if _, found := this.Ferry.Config.DataIterationBatchSizePerTableOverride[table.Schema]; !found {
+			this.Ferry.Config.DataIterationBatchSizePerTableOverride[table.Schema] = map[string]uint64{}
+		}
+		this.Ferry.Config.DataIterationBatchSizePerTableOverride[table.Schema][table.Name] = uint64(30)
+	}
+
+	for _, table := range this.tables {
+		this.Require().Equal(uint64(30), this.di.CursorConfig.GetBatchSize(table.Schema, table.Name))
+	}
+	this.Require().Equal(uint64(2), this.di.CursorConfig.GetBatchSize("DNE", "DNE"))
```

Comment:
nit: can you change this to `this.Ferry.Config.DataIterationBatchSize` so we know why it's "2"?